ide/coqide/preferences.ml

```diff
@@ -666,10 +666,47 @@ let tag_button () =
 let pref_file = Filename.concat (Minilib.coqide_config_home ()) "coqiderc"
 let accel_file = Filename.concat (Minilib.coqide_config_home ()) "coqide.keys"
 
+let accel_regex = Str.regexp {|\(; \|\)(gtk_accel_path "\([^""]*\)"|}
```

Comment:
Nit: This might just be my inability to parse regex, however I think adding a comment with an example of such a sentence might benefit readability.

Reply:
This is a routine use of a regex.  Regexes are clearly explained [here](https://v2.ocaml.org/api/Str.html#1_Regularexpressions), so I don't think a comment is necessary or really that helpful.  Comments that mention non-obvious details or that give higher-level context are more valuable.

But to answer your question:

```
\(; \|\)(gtk_accel_path "\([^""]*\)"
\(; \|\)                              - recognizes either "; " or "".
                                        \( ... \) are parentheses for the regex
                                        \| separates alternatives
        (gtk_accel_path               - recognizes "(gtk_accel_path "
                        "\([^""]*\)"  - recognizes a quoted sequence of characters,
                                        \( ... \) are used so I can get the matching substring for sorting
                                        [^""]*  = 0 or more of non-" characters  ^ negates the char class
                                        repeating " is benign as I explained
```